npc/re/instances/LastRoom.txt

```diff
@@ -0,0 +1,586 @@
+
+1@uns,139,41,3	script	Mark#room1	4_M_BLUEMAN,{
+	mes "[Mark]";
+	mes "Something doesn't feel right. It is better that we move carefully.";
+	cutin "bu_mark4.bmp",0;
+	close3;
+}
+
+1@uns,145,32,5	script	Alp#room1	4_M_BLACKMAN,{
+	mes "[Alp]";
+	mes "There are suspicious traces everywhere.";
+	mes "Some kind of heavy machine seems to have passed...";
+	cutin "bu_alp1.bmp",2;
+	close3;
+}
+
+1@uns,145,37,3	script	Tamarin#room1	4_M_TAMARIN,{
+	mes "[Tamarin]";
+	mes "Verity, are you alright?";
+	mes "You do not look good...";
+	cutin "ep143_taang.bmp",2;
+	close3;
+}
+
+1@uns,143,38,5	script	Du#room1	4_M_REDMAN,{
+	mes "[Du]";
+	mes "I have a bad feeling about this...";
+	cutin "bu_du1.bmp",2;
+	close3;
+}
+
+1@uns,143,39,5	script	Maggi#room1	4_F_PINKWOMAN,{
+	mes "[Maggi]";
+	mes "Aahh....";
+	cutin "bu_maggi4.bmp",2;
+	close3;
+}
+
+1@uns,144,37,3	script	Verity#room1	4_F_BERRYTEA,{
+	if ('protocole < 2) {
+		mes "[Verity]";
+		mes "Umph...I suddenly have a feeling that's not good.";
+		cutin "EP15_2_brt_6.bmp",2;
+		if (getpartyleader(getcharid(1),2) != getcharid(0)) {
+			mes "I wish to talk with the leader.";
+			close3;
+		}
+		npctalk "Verity: Umph...I suddenly have a feeling that's not good.";
+		next;
+		mes "[Verity]";
+		mes "Something is splitting my head....Ugh....";
+		npctalk "Verity: Something is splitting my head....Ugh....";
+		next;
+		mes "[Verity]";
+		mes "....Ugh....";
+		npctalk "Verity: ....Ugh....";
+		next;
+		mes "[Du]";
+		mes "Sister! Are you alright?!";
+		cutin "bu_du5.bmp",2;
+		npctalk "Du: Sister! Are you alright?!", instance_npcname("Du#room1");
+		next;
+		mes "[Verity]";
+		mes "....Have to go back....";
+		cutin "EP15_2_brt_7.bmp",2;
+		npctalk "Verity: ....Have to go back....";
+		next;
+		mes "[Tamarin]";
+		mes "Verity, are you alright?";
+		mes "Can you walk?";
+		cutin "ep143_tahuk.bmp",2;
+		npctalk "Tamarin: Verity, are you alright? Can you walk?", instance_npcname("Tamarin#room1");
+		next;
+		mes "[Verity]";
+		mes "Ugh...";
+		cutin "EP15_2_brt_6.bmp",2;
+		npctalk "Verity: Ugh...";
+		next;
+		mes "[Du]";
+		mes "" + strcharinfo(0) + "! I think it is better that you go back.";
+		cutin "bu_du5.bmp",2;
+		npctalk "Du: " + strcharinfo(0) + "! I think it is better that you go back.", instance_npcname("Du#room1");
+		next;
+		mes "[Verity]";
+		mes "N...no. I will catch up soon so go on ahead.";
+		cutin "EP15_2_brt_4.bmp",2;
+		npctalk "Verity: N...no. I will catch up soon so go on ahead.";
+		next;
+		mes "[Tamarin]";
+		mes "Then I will stay with Verity until she recovers so you can go on ahead.";
+		cutin "ep143_tahuk.bmp",2;
+		npctalk "Tamarin: Then I will stay with Verity until she recovers so you can go on ahead.";
+		close2;
+		'protocole = 1;
+		cutin "",255;
+	}
+	end;
+
+OnInstanceInit:
+	'map_name$ = instance_mapname("1@uns");
+	'protocole = 0;
+	disablenpc instance_npcname("protocole1");
+	disablenpc instance_npcname("Tamarin#room2");
+	disablenpc instance_npcname("zone_1_2_last_room");
```

Comment:
@Atemo boss, i cant find this npc "zone_1_2_last_room", do i miss something?

Reply:
i think thats a map name maybe need a new map for that